react-delegationdashboard/src/components/Overview/Cards/index.tsx

```diff
@@ -0,0 +1,190 @@
+import * as React from 'react';
+import { decimals, denomination } from 'config';
+import { useContext } from 'context';
+import { contractViews } from 'contracts/ContractViews';
+import denominate from 'components/Denominate/formatters';
+import StatCard from 'components/StatCard';
+import { ContractOverview, NodeType } from 'helpers/types';
+import { Query, QueryResponse } from '@elrondnetwork/erdjs/out/smartcontracts/query';
+import { Address, Argument, ContractFunction, NetworkStake } from '@elrondnetwork/erdjs/out';
+import { useState } from 'react';
+
+import SetPercentageFeeAction from './SetPercentageFeeAction';
+import UpdateDelegationCapAction from './UpdateDelegationCapAction';
+import AutomaticActivationAction from './AutomaticActivationAction';
+
+const Views = () => {
+  const { dapp, egldLabel, delegationContract, auctionContract, address } = useContext();
+  const { getTotalActiveStake, getContractConfig } = contractViews;
+  const [totalActiveStake, setTotalActiveStake] = React.useState('...');
+  const [noNodes, setNoNodes] = React.useState('...');
+  const [contractOverview, setContractOverview] = useState(new ContractOverview());
+  const [networkStake, setNetworkStake] = useState(new NetworkStake());
+  const [isAdminFlag, setIsAdminFlag] = useState(false);
+
+  const getContractOverviewType = (value: QueryResponse) => {
+    let delegationCap = denominate({
+      decimals,
+      denomination,
+      input: value.returnData[2].asBigInt.toString(),
+      showLastNonZeroDecimal: false,
+    });
+    let initialOwnerFunds = denominate({
+      decimals,
+      denomination,
+      input: value.returnData[3].asBigInt.toString(),
+      showLastNonZeroDecimal: false,
+    });
+    return new ContractOverview(
+      (parseFloat(value.returnData[1].asHex) / 100).toString(),
+      delegationCap,
+      initialOwnerFunds,
+      value.returnData[4]?.asString,
+      value.returnData[5].asBool,
+      value.returnData[6].asBool,
+      value.returnData[7].asBool,
+      value.returnData[8]?.asNumber * 6
+    );
+  };
+
+  const getPercentage = (firstValue: string, secondValue: string) => {
```

Comment:
I think we need a better naming for this two parameters

Reply:
done